crates/propolis-api-types/Cargo.toml

```diff
@@ -9,6 +9,7 @@ doctest = false
 
 [dependencies]
 crucible-client-types.workspace = true
+illumos-utils.workspace = true
```

Comment:
I don't see this used anywhere, is this a development change?

Reply:
Removed.